app/main.py

```diff
@@ -1 +1,8 @@
-# write your code here
+def copy_file(command: str) -> None:
+    files_names = command.split()
+    if files_names[1] == files_names[2]:
+        return
+    with open(f"{files_names[1]}", "r") as file_in, \
+            open(f"{files_names[2]}", "w") as file_out:
```

Comment:
what are files_names[1], files_names[2]?
create separated variables for their values